src/odb/include/odb/db.h

```diff
@@ -3227,24 +3238,30 @@ class dbInst : public dbObject
   /// If false, it will be added to the top module.
   /// Returns nullptr if an instance with this name already exists.
   /// Returns nullptr if the master is not FROZEN.
-  ///
+  /// If dbmodule is non null the dbInst is added to that module.
+
   static dbInst* create(dbBlock* block,
                         dbMaster* master,
                         const char* name,
-                        bool physical_only = false);
+                        bool physical_only = false,
+                        dbModule* = nullptr);
```

Comment:
Name the argument (eg parent_module).  Align with the comment.

Reply:
done. Note with interest that there are two dbInst create. Almost certainly wrong. Something to do with regions.

Reply:
They could be combined as one just calls the other with a null region.  I don't think I'll change this while this PR is in flight.